docs/plugins.md

```diff
@@ -1525,6 +1525,17 @@ These plugins provide new markers or news ways of converting abstract data into
 			<a href="https://github.com/Oliv">Olivier Gasc</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/JackZouShao/leaflet-marker-drection">leaflet-marker-drection</a>
```

Comment:
Sorry I did not notice previously, but are you sure you want to keep the typo in your plugin and repo name? ("drection" instead of "d**i**rection")

Reply:
.. sorry I'll rebuild a repository right away and comment

Reply:
yes, I had change  the name of your plugin a few  minutes ago .Delete old repository and built a new one  (T＿T)

Reply:
yes, I had change the name of my plugin a few minutes ago .Delete old repository and built a new one (T＿T)

Reply:
This is my first time using GitHub is also the first development leaflet plug-ins, I made some mistakes  but still completed successfully.At last,  thanks you for your help.  @ghybs  @cherniavskii